.markdownlint.yml

```diff
@@ -0,0 +1,4 @@
+default: true
+line-length: false
+no-inline-html: false
+first-line-heading: false # the README.md file has not a heading
```

Comment:
Would personally add an exception in the README file, instead of disabling this rule completely.